posthog/celery.py

```diff
@@ -364,9 +375,22 @@ def pre_run_signal_handler(task_id, task, **kwargs):
     tag_queries(kind="celery", id=task.name)
     set_default_clickhouse_workload_type(Workload.OFFLINE)
 
+    task_timings[task_id] = time.time()
```

Comment:
Might make sense to use `time.monotonic()` for both time calls.

Reply:
(Replying to your other comment) - the metric has the task_name so we can use that for filtering

Reply:
Ah, it's right there 👀 
That makes sense 🙂